### ISSUE TITLE
Make test_choose_representative_colors_different_values deterministic

### ISSUE THREAD

**Issue by lukasbindreiter** [lukasbindreiter/white-brush#20]
Sometimes the test `test_choose_representative_colors_different_values` (in `tests/test_calc_colors.py`) passes, sometimes it fails.

If you add another test method running that test often:
```python
def test_choose_representative_colors_different_values_often(self):
    for i in range(100):
        self.test_choose_representative_colors_different_values()
```
it will never pass

@p-hofer please take a look

**Comment by lukasbindreiter** [comment]
After debugging I found out that:
Sometimes the first and second row of the result are swapped:

**Expected output:**
![expected](https://i.imgur.com/IjbfRXl.jpg)


**Actual output:**
![actual](https://i.imgur.com/OwLHbBI.jpg)

**Comment by p-hofer** [comment]
I'm already on it, pushing the fix in a few minutes

**Comment by lukasbindreiter** [comment]
Resolved in #21
